src/ocl/tensorocl.cpp

```diff
@@ -239,16 +254,20 @@ void OpTensor3d(const Handle& handle,
                        miopen_beta,
                        long(Aoffset),
                        long(Boffset),
-                       long(Coffset));
+                       long(Coffset),
+                       long(total_work),
+                       int(!float_equal(miopen_alpha0, 0.0)),
```

Comment:
could we have used short in place of int to pass bool?

Reply:
same as above